src/main/java/uk/gov/hmcts/reform/sendletter/logging/ErrorCode.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.sendletter.logging;
+
+@SuppressWarnings({"squid:S1118", "HideUtilityClassConstructor"})
+public final class ErrorCode {
```

Comment:
This is going to be repeated every time I guess.Best to extract in logging project itself.May be a separate PR.

Reply:
yes. should definitely speak about it tomorrow or next tuesday